src/test/lambda/commands/deploySamApplication.test.ts

```diff
@@ -126,7 +127,15 @@ describe('deploySamApplication', async function () {
     let templatePath: string
 
     let tempToolkitFolder: string
+
+    // Fake "aws.refreshAwsExplorer" command. 50b5a28b8e35 #1665
+    let didCommand_aws_refreshAwsExplorer = false
+    vscode.commands.registerCommand('aws.refreshAwsExplorer', async () => {
+        didCommand_aws_refreshAwsExplorer = true
+    })
```

Comment:
Consider restoring the command in the `after` section since this isn't constrained to this test.

Reply:
Yeah. Technically the command didn't exist (which is why I needed this)... not sure what the best approach is. Could delete the command in after()... 

I considered init'ing awsExplorer in globalSetup, but that introduces different questions...

Reply:
I could see other tests using this in the future...maybe turn it into a general test util? Could probably be good for other commands too.

Reply:
Could make a 'sandbox' for VS Code commands as a disposable. Not sure if you can get the currently registered command though as an object. Would be needed to restore functionality.